src/Schemas/Model/Model.php

```diff
@@ -145,7 +144,7 @@ public function getLabelKey(): ?Attribute
 
     public function getAliases(): Collection
     {
-        return $this->attributes->filter(fn (Attribute $attribute) => $attribute->getType() === AttributeType::ALIAS);
+        return $this->attributes->filter(fn (Attribute $attribute) => $attribute->getIsAlias());
```

Comment:
@ShreyDhyani Rename this to isAliasKey()

Reply:
Removed function `getIsAlias ` and used `isAliasKey` for all occurances.